workers/php/types/php-event-worker-task.d.ts

```diff
@@ -0,0 +1,17 @@
+import { EventWorkerTask } from '../../../lib/types/event-worker-task';
+import { DecodedEventData, EventAddons } from 'hawk.types';
+
+/**
+ * Describe a context passed from Php Catcher
+ */
+interface PhpEventPayload extends DecodedEventData<EventAddons> {}
```

Comment:
There should be `PhpAddons`. Described [there](https://github.com/codex-team/hawk.types/blob/master/src/base/event/addons/php.ts)